proto/com/iabtechlab/adcom/v1/enums/enums.proto

```diff
@@ -692,3 +692,13 @@ enum VolumeNormalizationMode {
   LOUDNESS_NORMALIZATION = 3;  // Ad Loudness Normalized to Content.
   VOL_CUST_NORMALIZATION = 4;  // Custom Volume Normalization.
 }
+
+/**
+ *  The following enum lists the possible sources for User-Agent data.
+ */
+enum UserAgentSource {
+  UNKNOWN_SOURCE            = 0;
```

Comment:
For the "unknown" values, please apply the convention of `<enum_name>_UNKNOWN`, i.e., `UserAgentSource_UNKNOWN`.